.github/workflows/build-docker-image-to-prod.yml

```diff
@@ -58,9 +61,10 @@ jobs:
           context: backend
           tags: |
             infisical/backend:${{ steps.commit.outputs.short }}
-            infisical/backend:latest
             infisical/backend:${{ steps.extract_version.outputs.version }}
           platforms: linux/amd64,linux/arm64
+          labels: |
+            version=${{ steps.extract_version.outputs.version }}
```

Comment:
I don't think we need this. Any reason you added labels ?

Reply:
just metadata on the image, agree it's not needed